src/next-version-pr.ts

```diff
@@ -0,0 +1,88 @@
+import { cdk } from "projen";
+import { GithubWorkflow } from "projen/lib/github";
+import { JobPermission } from "projen/lib/github/workflows-model";
+
+const PULL_REQUEST_REF = "${{ github.event.pull_request.head.ref }}";
+const PULL_REQUEST_REPOSITORY =
+  "${{ github.event.pull_request.head.repo.full_name }}";
+
+export class NextVersionPr {
+  constructor(project: cdk.JsiiProject, githubSecret: string) {
+    const workflow = new GithubWorkflow(
+      project.github!,
+      "next-cdktf-version-pr"
+    );
+    workflow.on({ workflowDispatch: {} });
+    workflow.addJob("pr-against-next", {
+      runsOn: ["ubuntu-latest"],
+      permissions: {
+        contents: JobPermission.WRITE,
+        pullRequests: JobPermission.WRITE,
+      },
+      env: {
+        CI: "true",
+        CHECKPOINT_DISABLE: "1",
+      },
+      steps: [
+        {
+          name: "Checkout",
+          uses: "actions/checkout@v3",
+          with: {
+            ref: PULL_REQUEST_REF,
+            repository: PULL_REQUEST_REPOSITORY,
+          },
+        },
+        {
+          name: "Remove old PR",
+          run: [
+            `PR_NUMBER_TO_CLOSE=$(gh pr list | grep "cdktf-next-pr" | awk '{ print $1 }')`,
+            `if [ -z "$PR_NUMBER_TO_CLOSE" ]; then`,
+            `  echo "No PR to close"`,
+            `else`,
+            `  gh pr close $PR_NUMBER_TO_CLOSE`,
+            `fi`,
+          ].join("\n"),
+          env: {
+            GH_TOKEN: githubSecret,
+          },
```

Comment:
Can't it update the existing PR instead? What is the reason to close and create a new one? (It feels more noisy that way, but idk may be perfectly fine)

Reply:
ah, fair enough 👍